### DISCUSSION TITLE
Same speed for all layer

### DISCUSSION THREAD

**Discussion by Pgmiranda1** [supermerill/SuperSlicer#2221] [Community contribution]
How to configure for all layers (except the first) to have the same speed?
![image](https://user-images.githubusercontent.com/94395539/149432403-88e2d176-58a6-4913-8d35-0c88a96bb765.png)
![image](https://user-images.githubusercontent.com/94395539/149432444-ff548b42-a56f-4b60-a893-6ed98334d7d2.png)

**Comment by neophyl** [comment]
Without a copy of a saved project file its impossible to say why your settings aren't producing your 50mm/s speed.  When I configure mine to the same settings you have pictured I get 50mm/s.
![image](https://user-images.githubusercontent.com/46955527/149473299-f94db511-0a36-484e-9c62-cd6c1de0e512.png)

If I had to guess I would say your filament cooling settings are coming into play.  However thats a guess.  The point of saving a zipped up 3mf project is people can load it and then dont have to guess.  There are hundreds of settings, many of them interact in ways that are not obvious and posting pics of a few settings is not the most efficient way to get help.
